blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_2.py

```diff
@@ -22,11 +23,11 @@ def __init__(
 
         Parameters
         ----------
-        theta_i : float
-            [Θi] Eccentricity [m].
-        theta_i : float
-            [Θi] Initial inclination imperfections, Θi, is a ratio between height and inclination of the member [-].
+        theta_i : DIMENSIONLESS
+            [Θi] Eccentricity, initial inclination imperfections [-].
```

Comment:
how do we handle subscript in docstrings?

Reply:
Good question. It's on my list ;-)